.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+**Before submitting a pull request,** please make sure the following is done:
```

Comment:
This is to avoid PRs without unit tests (more or less), and with obvious performance pitfalls.

If a contributor has put the effort so far, it means that they want to!